be/src/olap/page_cache.cpp

```diff
@@ -0,0 +1,53 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "olap/page_cache.h"
+
+namespace doris {
+
+// This should only be used in unit test. 1GB
+static StoragePageCache s_ut_cache(1073741824);
```

Comment:
Specially handle for UT is necessary?

Reply:
It make it easy to write unit test for class which depends on this class. Otherwise, developer  should call create_global_instance(). And the cost of having an unused object is acceptable.

Reply:
OK.